### PR TITLE
LS-457-starlight-doc

### PR DESCRIPTION
**Pull request by mendonk**
Doc on Starlight and links to RabbitMQ, JMS, and Kafka docs.

### CODE REVIEW

**Review by eolivelli — APPROVED** [review]

**Review by nicoloboschi — APPROVED** [review]

File: modules/ROOT/pages/starlight.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Starlight for JMS allows enterprises to take advantage of the scalability and resiliency of a modern streaming platform to run their existing JMS applications. Because Apache Pulsar™ is open-source and cloud-native, Starlight for JMS enables enterprises to move their JMS applications to run on-premises and in any cloud environment.
+
+To get started with Starlight for JMS in Luna Streaming, check out 
+https://docs.datastax.com/en/fast-pulsar-jms/docs/1.1/pulsar-jms-install.html[Starlight for JMS Quickstart].
+
+== Starlight for Kafka
+
+Starlight for Kafka allows your Apache Kafka® clients to use an Apache Pulsar® cluster.
+
+To get started with Starlight for Kafka in Luna Streaming, check out the https://github.com/datastax/starlight-kafka-docs[Starlight for Kafka repo].
```
Comment (jgillenwater) [inline modules/ROOT/pages/starlight.adoc:27]:
```suggestion
// == Starlight for Kafka
// 
// Starlight for Kafka allows your Apache Kafka® clients to use an Apache Pulsar® cluster.
// 
// To get started with Starlight for Kafka in Luna Streaming, check out the https://github.com/datastax/starlight-kafka-docs[Starlight for Kafka repo]. 
```

File: modules/ROOT/pages/starlight.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Starlight for RabbitMQ™ is open source and is included in DataStax’s Luna Streaming Enterprise support for Pulsar. 
+
+For implementation with JMS or Kafka on a Pulsar cluster, we also offer Starlight for JMS and Starlight for Kafka 
+
+== Starlight for RabbitMQ
+
+Starlight for RabbitMQ™ is open source and is included in DataStax’s Luna Streaming Enterprise support for Pulsar.
+
+For more information on Starlight for Rabbit MQ, check out 
+https://docs.datastax.com/en/starlight-rabbitmq/docs/1.0/getting-started.html[Starlight for Rabbit MQ Quickstart].
```
Comment (jgillenwater) [inline modules/ROOT/pages/starlight.adoc:14]:
```suggestion
= Starlight for Luna Streaming

Starlight for Luna Streaming allows enterprises to take advantage of the scalability and resiliency of a modern streaming platform with various connectors for Apache Pulsar™.

== Starlight for RabbitMQ

Starlight for RabbitMQ™ combines the industry-standard AMQP 0.9.1 (RabbitMQ) API with the cloud-native and horizontally scalable Apache Pulsar™ streaming platform, providing a powerful way to modernize your RabbitMQ infrastructure, improve performance, and reduce costs.

Starlight for RabbitMQ™ is open source and is included in DataStax’s Luna Streaming Enterprise support for Pulsar. 

For more information on Starlight for Rabbit MQ, check out 
https://docs.datastax.com/en/starlight-rabbitmq/docs/1.0/getting-started.html[Starlight for Rabbit MQ Quickstart].
```

**Review by jgillenwater — APPROVED** [review]

File: modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -7,4 +7,3 @@
 *** xref:heartbeat-vm.adoc[Installing Heartbeat on Server/VM]
 *** xref:pulsar-monitor.adoc[Monitoring Luna using Pulsar Heartbeat]
 *** xref:faqs.adoc[Luna Streaming FAQs]
-** xref:admin-console-VM.adoc[Install Admin Console on Server/VM]
```
Comment (jgillenwater) [inline modules/ROOT/nav.adoc:10]: Why was this removed as part of the PR? Also, should `starlight.adoc` be added to the nav?

File: modules/ROOT/pages/starlight.adoc [hunk 4]
```diff
@@ -0,0 +1,25 @@
+= Starlight for Luna Streaming
+
+Starlight for Luna Streaming allows enterprises to take advantage of the scalability and resiliency of a modern streaming platform with various connectors for Apache Pulsar™.
+
+== Starlight for RabbitMQ
+
+Starlight for RabbitMQ™ combines the industry-standard AMQP 0.9.1 (RabbitMQ) API with the cloud-native and horizontally scalable Apache Pulsar™ streaming platform, providing a powerful way to modernize your RabbitMQ infrastructure, improve performance, and reduce costs.
```
Comment (jgillenwater) [inline modules/ROOT/pages/starlight.adoc:7]:
```suggestion
Starlight for RabbitMQ™ combines the industry-standard AMQP 0.9.1 (RabbitMQ) API with the cloud-native and horizontally scalable Pulsar streaming platform, providing a powerful way to modernize your RabbitMQ infrastructure, improve performance, and reduce costs.
```

File: modules/ROOT/pages/starlight.adoc [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+Starlight for RabbitMQ™ is open source and is included in DataStax’s Luna Streaming Enterprise support for Pulsar. 
+
+For more information on Starlight for Rabbit MQ, check out 
+https://docs.datastax.com/en/starlight-rabbitmq/docs/1.0/getting-started.html[Starlight for Rabbit MQ Quickstart].
```
Comment (jgillenwater) [inline modules/ROOT/pages/starlight.adoc:12]:
```suggestion
For more information on Starlight for RabbitMQ, check out 
https://docs.datastax.com/en/starlight-rabbitmq/docs/1.0/getting-started.html[getting started with Starlight for RabbitMQ].
```

File: modules/ROOT/pages/starlight.adoc [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+== Starlight for JMS
+
+Starlight for JMS allows enterprises to take advantage of the scalability and resiliency of a modern streaming platform to run their existing JMS applications. Because Apache Pulsar™ is open-source and cloud-native, Starlight for JMS enables enterprises to move their JMS applications to run on-premises and in any cloud environment.
```
Comment (jgillenwater) [inline modules/ROOT/pages/starlight.adoc:16]:
```suggestion
Starlight for JMS allows enterprises to take advantage of the scalability and resiliency of a modern streaming platform to run their existing JMS applications. Because Pulsar is open-source and cloud-native, Starlight for JMS enables enterprises to move their JMS applications to run on-premises and in any cloud environment.
```

File: modules/ROOT/pages/starlight.adoc [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+To get started with Starlight for JMS in Luna Streaming, check out 
+https://docs.datastax.com/en/fast-pulsar-jms/docs/1.1/pulsar-jms-install.html[Starlight for JMS Quickstart].
```
Comment (jgillenwater) [inline modules/ROOT/pages/starlight.adoc:19]:
```suggestion
To get started with Starlight for JMS in Luna Streaming, check out 
https://docs.datastax.com/en/fast-pulsar-jms/docs/1.1/pulsar-jms-install.html[Starlight for JMS installation].
```

File: modules/ROOT/pages/starlight.adoc [hunk 8]
```diff
@@ -0,0 +1,25 @@
+= Starlight for Luna Streaming
+
+Starlight for Luna Streaming allows enterprises to take advantage of the scalability and resiliency of a modern streaming platform with various connectors for Apache Pulsar™.
+
+== Starlight for RabbitMQ
+
+Starlight for RabbitMQ™ combines the industry-standard AMQP 0.9.1 (RabbitMQ) API with the cloud-native and horizontally scalable Apache Pulsar™ streaming platform, providing a powerful way to modernize your RabbitMQ infrastructure, improve performance, and reduce costs.
+
+Starlight for RabbitMQ™ is open source and is included in DataStax’s Luna Streaming Enterprise support for Pulsar.
```
Comment (jgillenwater) [inline modules/ROOT/pages/starlight.adoc:9]: Does this apply to Starlight for JMS? What about when Starlight for Kafka is added? If all of the Starlight connectors are included in the support plan, let's move this to the first section and make it more generic. Let's also link to the support policy or another Luna Streaming reference similar. Probably https://www.datastax.com/legal/luna-streaming-supplemental-support-policy, but confirm with @chrislatimer.

**Review by jgillenwater — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/starlight.adoc (hunk 8 above)
Comment (jgillenwater) [reply modules/ROOT/pages/starlight.adoc]: @mendonk Let's also link to the support policy or another Luna Streaming reference similar. Probably https://www.datastax.com/legal/luna-streaming-supplemental-support-policy, but confirm with @chrislatimer.
